src/storage-folder.coffee

```diff
@@ -0,0 +1,27 @@
+path = require "path"
+fs = require "fs-plus"
+
+module.exports =
+class StorageFolder
+  constructor: (containingPath) ->
+    @path = path.join(containingPath, "storage")
+
+  store: (name, object) ->
+    fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
+
+  load: (name) ->
+    try
+      stateString = fs.readFileSync(@pathForKey(name), 'utf8')
+    catch error
+      if error.code is 'ENOENT'
+        return undefined
+      else
+        throw error
```

Comment:
Looks like Atom fails to start if this error is thrown, should it just resort to the default window in this case instead?